packages/plugin/src/rules/lone-executable-definition.ts

```diff
@@ -24,7 +24,7 @@ const uppercaseFirst = (string: string) => string.slice(0, 1).toUpperCase() + st
 const isDefinitionNode = (node: { kind: string }): node is DefinitionESTreeNode =>
   node.kind === Kind.OPERATION_DEFINITION || node.kind === Kind.FRAGMENT_DEFINITION;
 
-const rule: GraphQLESLintRule<[LoneExecutableDefinitionConfig]> = {
+export const rule: GraphQLESLintRule<[LoneExecutableDefinitionConfig]> = {
```

Comment:
It seems that this change (named instead of default export) is breaking the tests.

Reply:
Yeah, I already fixed it, made some refactoring to use ESLint selectors, you should like it